minitron.rb

```diff
@@ -51,6 +51,8 @@
       timestamp: ticket.created_at,
       properties: {
         ticket_id: id,
+        requester_name: ticket.requester.name,
```

Comment:
This doesn't belong in the event properties. Names should get tracked as part of an `identify` call, which happens earlier.